types/conf.go

```diff
@@ -164,18 +168,26 @@ func mergeCNIRuntimeConfig(runtimeConfig *RuntimeConfig, delegate *DelegateNetCo
 		if delegate.DeviceID != "" {
 			runtimeConfig.DeviceID = delegate.DeviceID
 		}
+		logging.Debugf("mergeCNIRuntimeConfig: add runtimeConfig for net-attach-def: %v", runtimeConfig)
 	}
 
 	return runtimeConfig
 }
 
 // CreateCNIRuntimeConf create CNI RuntimeConf for a delegate
```

Comment:
This function has changed, maybe update the comment?

Reply:
Tweaked the comment. Let me know if you think there should be more.